src/pages/SubmissionConfig.js

```diff
@@ -370,6 +372,151 @@ const renderGrid = props => {
   }])
 }
 
+
+const renderInputTable = ({
+  selectedDataTable,
+  method,
+  configuredInputDefinition, setConfiguredInputDefinition,
+  cachedInputSources, setCachedInputSources,
+  inputTableSort, setInputTableSort
+}) => {
+  const dataTableAttributes = _.keyBy('name', selectedDataTable.attributes)
+
+  const inputSourceLabels = {
+    default: 'Use Default',
+    literal: 'Type a Value', // TODO
+    record_lookup: 'Fetch from Data Table'
+  }
+  const inputSourceTypes = _.invert(inputSourceLabels)
+
+  const parseInputType = inputType => {
+    const { primitive_type: primitiveType, optional_type: optionalType } = inputType
+    return primitiveType ? primitiveType : `${optionalType.primitive_type} (optional)`
+  }
+
+  const recordLookupSelect = rowIndex => {
+    return h(Select, {
+      isDisabled: false,
+      'aria-label': 'Select an Attribute',
+      isClearable: false,
+      value: _.get(`${rowIndex}.source.record_attribute`, configuredInputDefinition) || _.get(`${rowIndex}.record_attribute`, cachedInputSources),
+      onChange: ({ value }) => {
+        const newAttribute = _.get(`${value}.name`, dataTableAttributes)
+        const newSource = {
+          type: _.get(`${rowIndex}.source.type`, configuredInputDefinition),
+          record_attribute: newAttribute
+        }
+        const newConfig = _.set(`${rowIndex}.source`, newSource, configuredInputDefinition)
+        setConfiguredInputDefinition(newConfig)
+        setCachedInputSources(_.set(`${rowIndex}.record_attribute`, newSource.record_attribute, cachedInputSources))
+      },
+      placeholder: 'Select Attribute',
+      options: _.keys(dataTableAttributes),
+      // ** https://stackoverflow.com/questions/55830799/how-to-change-zindex-in-react-select-drowpdown
+      styles: { container: old => ({ ...old, display: 'inline-block' }), menuPortal: base => ({ ...base, zIndex: 9999 }) },
+      menuPortalTarget: document.body,
+      menuPlacement: 'top'
```

Comment:
this crudely solves an annoying problem: the rows at the bottom of the inputs selector table don't have enough space to accommodate the dropdown menus. I tried to make space so that the user could scroll down, but in the end everything I tried failed.

This solution looks a bit odd (have all dropdowns "drop up"), but it works... improving this should be considered a "nice to have", in my opinion